about.php

```diff
@@ -1,51 +1,60 @@
-<?php require_once __DIR__ .'/variables.php' ?>
+<?php
+require_once __DIR__ . '/variables.php';
+?>
+
 <!DOCTYPE html>
 <html lang="en">
-<?php require_once __DIR__ .'/template/head.phtml';?>
-<title><?php $titleAbout?></title>
+<?php require_once __DIR__ . '/template/head.phtml'; ?>
+<?php
+require_once __DIR__ . '/template/title.phtml';
+?>
+<title><?php echo $title = getPageTitle($page_titles); ?></title>
 <body>
-   <header>
-       <?php require_once __DIR__ .'/template/menu.phtml';?>
-    <ul class="menu-nav">
-        <?php
-        foreach ($menuitems as $url => $label){
-        ?>
-        <li class="nav-item ">
-            <a href="<?=$url?>" class="nav-link"><?=$label?></a>
+<header>
+    <nav>
+        <?php require_once __DIR__ . '/template/menu.phtml'; ?>
+        <ul class="menu-nav">

```

Comment:
@ReginaBorisova  Isn't this part repeats in every file? Can it go to the menu.phtml?

Reply:
moved to menu.phtml